flink-runtime/src/main/java/org/apache/flink/runtime/rest/RestClient.java

```diff
@@ -229,7 +229,7 @@ public void shutdown(Time timeout) {
 			Collection<FileUpload> fileUploads,
 			RestAPIVersion apiVersion) throws IOException {
 		Preconditions.checkNotNull(targetAddress);
-		Preconditions.checkArgument(0 <= targetPort && targetPort < 65536, "The target port " + targetPort + " is not in the range (0, 65536].");
+		Preconditions.checkArgument(0 <= targetPort && targetPort < 65536, "The target port " + targetPort + " is not in the range [0, 65536).");
```

Comment:
why not do it as precode, eg:
		Preconditions.checkArgument(0 <= targetPort && targetPort <= 65535, "The target port " + targetPort + " is not in the range [0, 65535].");

Reply:
well, I will take your advice